skt/reporter.py

```diff
@@ -295,9 +295,10 @@ def getmergeinfo(self):
             result += ['  - %s,' % pname,
                        '    grabbed from %s\n' % purl]
 
-        result += ['on top of commit %s from' % self.mergedata['base'][1][:12],
-                   '  %s' % self.mergedata['base'][0],
-                   'repository.']
+        result += ['on top of commit %s from the repository at' %
+                   self.mergedata['base'][1][:12],
+                   '  %s' %
+                   self.mergedata['base'][0]]
```

Comment:
Can you put this together with the previous line? Technically there's nothing wrong with it, but since it logically belongs together and fits < 80 chars it would be cleaner.

Reply:
Yeah, I changed it to be more readable for me, but your way is alright too :) Done!